$NetBSD: patch-ao,v 1.2 2010/06/12 19:36:01 dholland Exp $

Print and parse time_t as intmax_t, not as long (which might not fit).

--- src/docmd.c.orig	2001-03-12 18:42:23.000000000 +0000
+++ src/docmd.c
@@ -786,7 +786,7 @@ static void dodcolon(cmd, filev)
 		return;
 	}
 
-	debugmsg(DM_MISC, "%s: mtime %d\n", stamp, stb.st_mtime);
+	debugmsg(DM_MISC, "%s: mtime %jd\n", stamp, (intmax_t)stb.st_mtime);
 
 	env = NULL;
 	for (sc = sbcmds; sc != NULL; sc = sc->sc_next) {
